scripts/aggregate-themes/README.md

```diff
@@ -57,8 +60,61 @@ The `exclude` option accepts an array of files to exclude from being aggregated.
 
 #### Theme (Optional)
 
-The `theme` option accepts the string name of a default theme. The default theme will be applied directly to the application. If a `root-theme.scss` is found only that single file will be aggregated.
+The `theme` option accepts the string name of a default theme. The default theme will be applied directly to the application.
 
 #### Scoped (Optional)
 
-The `scoped` option accepts an array of theme names to aggregate. Only the `scoped-theme.scss` files will be aggregated.
\ No newline at end of file
+The `scoped` option accepts an array of theme names to generate or aggregate.
+
+## Example
+This example follows the [sample project structure](#Sample-Project-Structure) defined above.
+
+`${themeName}` files should contain imports or CSS custom property values encased in a `:global` scope.
+### themes/terra-dark-theme/terra-dark-theme.scss
+```scss
+:global {
+  @import 'component-1';
+  @import 'component-2';
+}
+```
+
+Using the above `terra-theme-config` in conjunction with the above project structure generates:
+
+### generatedThemes/root-terra-dark-theme.scss
+```scss
+:root {
+  @import '../node_modules/terra-component/themes/terra-dark-theme/terra-dark-theme.scss';
+  @import '../themes/terra-dark-theme/terra-dark-theme.scss';
+}
+```
+
+### generatedThemes/scoped-terra-light-theme.scss
+```scss
+.light-theme {
+  @import '../node_modules/terra-component/themes/terra-light-theme/theme-variables.scss';
+}
+```
+
+### generatedThemes/aggregated-themes.js
+```scss
+import './root-terra-dark-theme.scss';
+import './scoped-terra-light-theme.scss';
+```
+## Getting Started - Using Existing Root and Scoped Themes
```

Comment:
I would title this `Legacy` or something similar, and tag it as being deprecated and scheduled for removal in a future major release.

Reply:
https://github.com/cerner/terra-toolkit/pull/324/commits/882454cae629945311b47d43e6e6fe78c8cd6f45